scripts/update-ndc-postgres.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Bash doesn't always live in `/bin`.

```suggestion
#!/usr/bin/env bash
```